src/sass/_upload_instance_modal.scss

```
@@ -1,6 +1,17 @@
 .upload-instance-modal {
-  .p-modal__dialog {
-    margin: auto;
-    width: 35rem;
+  .fixed-height {
+    height: 35rem;
+  }
+
+  .content-details {
+    padding-left: $sph--x-small;
```

```suggestion
    padding-left: $sph--x-small;
    margin-left: -$sph--x-small;
```
Better keep the content aligned with the title in the header

---

neat!